### ISSUE TITLE
Feed missing

### ISSUE THREAD

**Issue by timpool06** [Drumber/Kitsune#65]
### Description

One of my favorite aspects of Kitsu is the feed. I can see posts from people/groups I have followed, it makes Kitsu feel less like an anime tracker and more like a social network for anime fans. This app doesn't have a feed, at all. It's literally just an anime tracker, no way to see or interact with other users.

### Acknowledgements

- [X] I have searched the open and closed [issues](https://github.com/Drumber/Kitsune/issues?q=is%3Aissue) and this is **NOT** a duplicate.

**Comment by Drumber** [comment]
Yeah, that's true. Right now, the app mainly focuses on tracking and searching for anime/manga.
I've been planning to add social features, like reactions and feeds. But first, I wanted to build a solid app for tracking and searching for anime/manga on Kitsu.

I know the app doesn’t provide the full "Kitsu experience," including interactions with other users. In its current state, the app isn’t meant to be a replacement for the official Kitsu app or website; it's more of a companion.

Adding social features will be the next major milestone for Kitsune. But I don't know when I'll have time, so I can’t make any promises yet.

**Comment by nubesurrealista** [comment]
I have a small suggestion regarding this issue that could serve as a temporary solution.

Address the feed issue with a minimalist solution: **an option to open the feed in webview**. This way you wouldn't have full feed integration but you would have quick access. You can look at the Al-Chan app (AniList client) for example, which displays the activity feed from that platform in a webview window.

**Comment by Drumber** [comment]
Feeds are now available in release v2.1.0
